pycolmap/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.10)
+project(${SKBUILD_PROJECT_NAME} VERSION ${SKBUILD_PROJECT_VERSION})
+
+set(CMAKE_CUDA_ARCHITECTURES "native")
```

Comment:
Do we really want to hardcode this to native? This prevents us from build redistributable binaries (or wheels) that run on other architectures than the current machine?

Reply:
Could we instead make it the default in `FindDependencies.cmake`? Most users that compile from source do so for their own machine only so they don't need to known about this flag.
https://github.com/colmap/colmap/blob/fb09740a390ea01806dc3945c18d031fa7814145/cmake/FindDependencies.cmake#L124-L128
```diff
    if(NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
-        message(
-            FATAL_ERROR "You must set CMAKE_CUDA_ARCHITECTURES to e.g. 'native', 'all-major', '70', etc. "
-            "More information at https://cmake.org/cmake/help/latest/prop_tgt/CUDA_ARCHITECTURES.html")
+        set(CMAKE_CUDA_ARCHITECTURES "native")
    endif()
```

Reply:
Sure, we can.

Reply:
Done, updated the doc as well.